### ISSUE TITLE
Rule for values inside object?

### ISSUE THREAD

**Issue by gellerb** [mozilla/scanjs#176]
Is it possible to write a rule to detect that `bar` is set to `'string'` in the below object? Ideally, I would like to detect both below use cases?

```
jsvar dboptions = {
    foo: {
        bar: 'string'
    }
}
```
or
`db.connect(dbconnect, { foo: { bar: 'string'} } )`

**Comment by mozfreddyb** [comment]
> Is it possible to write a rule to detect that foo is to to bar inside a given object?

Sorry, but I don't understand what you want to detect. Can you rephrase?

**Comment by dpnishant** [comment]
Hi @mozfreddyb ,

To me, it seems @gellerb is trying to detect if the "source" or "sink" is an attribute of an object.

**Comment by gellerb** [comment]
I'm new to ESLint but are the above test cases better suited for ESLint?
